kubernetes/couchdb/README.md

```diff
@@ -9,7 +9,7 @@ The db.auth secret and db.config configmap contain authorization and
 configuration information for the CouchDB instance being used for this
 OpenWhisk deployment.  The db.auth secret is expected to define two
 keys: db_username and db_password. The db.config configmap is expected
-to define five keys: db_protocol, db_provider, db_prefix,
+to define seven keys: db_protocol, db_provider, db_host, db_port, db_prefix,
```

Comment:
maybe should just say "define the following keys"

Reply:
👍 done.